kyaml/fn/runtime/runtimeutil/functiontypes.go

```diff
@@ -119,3 +135,52 @@ func getFunctionSpecFromAnnotation(n *yaml.RNode, meta yaml.ResourceMeta) *Funct
 	_ = yaml.Unmarshal([]byte(s), &fs)
 	return &fs
 }
+
```

Comment:
// StringToStorageMount converts a string like ??? to an instance of StorageMount.
Maybe better as a `fromString` factory method in the `StorageMount` type.